.github/workflows/release.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   release:
     name: Release
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
```

Comment:
Also considered using `ubuntu-latest`. I assumed an explicit operating system version was chosen for safety, and proceeded according to that assumption.